### ISSUE TITLE
Publish the language server

### ISSUE THREAD

**Issue by pantharshit00** [abelljs/vscode-abell-language-features#4]
We should publish the language server so that people can make integrations for other editors without rewriting all this logic.

I would personally like to make a coc.nvim extension for this.

**Comment by saurabhdaware** [comment]
By publishing do you mean publishing to microsoft marketplace? because it is supposed to be there already.

**Comment by pantharshit00** [comment]
@saurabhdaware No, I am talking about publishing the language server as an npm package.

Here is how we do that at work: https://www.npmjs.com/package/@prisma/language-server
https://github.com/prisma/language-tools/tree/master/packages/language-server

VSCode is one of the consumer of this language server.

**Comment by saurabhdaware** [comment]
Ohhh cool coool. Yes I will do this
